### PR TITLE
2.8 Hot Fix EBS-K8s-Permissions

### PR DESCRIPTION
**Pull request by resmaa**
IAM policies update for EBS

### CODE REVIEW

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Review by brathina-spectro — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/846#pullrequestreview-1085955900" title="Approved">brathina-spectro</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/846#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-aws-ebs.md
Comment (sadysnaat) [inline content/docs/06-integrations/00-aws-ebs.md:50]: instead of this we can specify to use amazon managed policy `AmazonEBSCSIDriverPolicy`

Comment (sadysnaat) [reply content/docs/06-integrations/00-aws-ebs.md:50]: if user wishes to have encryption enabled 

{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "kms:GenerateDataKeyWithoutPlaintext",
                "kms:CreateGrant"
            ],
            "Resource": "*"
        }
    ]
}

use these permissions

**Review by sadysnaat — COMMENTED** [review]
please ref comments

File: content/docs/06-integrations/00-aws-ebs.md
Comment (sadysnaat) [reply content/docs/06-integrations/00-aws-ebs.md:37]: @brathina-spectro the idea is to specify these permissions are attached to cloud account 
and palette will internally take care of attaching them to 

	//                - 'control-plane.cluster-api-provider-aws.sigs.k8s.io'
	//                - 'nodes.cluster-api-provider-aws.sigs.k8s.io'
	
and setting them in roleAdditionalPolicies for eks clusters

File: content/docs/06-integrations/00-aws-ebs.md
Comment (resmaa) [reply content/docs/06-integrations/00-aws-ebs.md:43]: scenario removed

File: content/docs/06-integrations/00-aws-ebs.md
Comment (resmaa) [reply content/docs/06-integrations/00-aws-ebs.md:58]: > if user wishes to have encryption enabled
> 
> { "Version": "2012-10-17", "Statement": [ { "Sid": "VisualEditor0", "Effect": "Allow", "Action": [ "kms:GenerateDataKeyWithoutPlaintext", "kms:CreateGrant" ], "Resource": "*" } ] }
> 
> use these permissions

added

File: content/docs/06-integrations/00-aws-ebs.md
Comment (sadysnaat) [inline content/docs/06-integrations/00-aws-ebs.md:67]: we need to remove this

File: content/docs/06-integrations/00-aws-ebs.md
Comment (sadysnaat) [inline content/docs/06-integrations/00-aws-ebs.md:43]: this is valid if user doesn't want to add these permissions to aws cloud account 

then he will need to do this 

otherwise only adding policies at aws cloud account level will take care of both aws infra and eks automatically

**Review by sadysnaat — CHANGES_REQUESTED** [review]

**Review by sadysnaat — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/846#pullrequestreview-1085955900" title="Approved">brathina-spectro</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/846#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/846#pullrequestreview-1088166025" title="Approved">sadysnaat</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
